File: 8-RakamSayısınıBulmak/RakamSayisi.py
# “x” ve “i”nin tanımlanması
x = 0
i = 0

print("3 haneli sayi giriniz: ")

# “x”in girilmesi
x = int(input())

# 3 kere dönen döngünün oluşturulması
for i in range(0,3):
    #  Kalanın yazdırılması
    print(x % 10)

    # “x”in onlar basamağının birler basamağına dönüştürülmesi
    x = x / 10

File: 8-RakamSayısınıBulmak/RakamSayisi.cpp
#include <iostream>


using namespace std;

int main() {
	cout << "Uc haneli bir sayi giriniz : ";
	int sayi;
	cin >> sayi;
	for (int i = 0; i < 3; ++i) {
		cout << sayi % 10 << '\n';
		sayi = sayi / 10;
	}
}
